tcms/testplans/views.py

```diff
@@ -1005,7 +1005,7 @@ def link_cases(self, template_name='plan/search_case.html'):
                                       context_instance=RequestContext(request))
 
         def delete_cases(self):
-            if not request.GET.get('case'):
+            if not request.POST.get('case'):
```

Comment:
The entire `delete_cases` method is not covered via tests. Also this is a class, defined inside the `cases` method, which allows both GET and POST. Either `delete_cases` needs to require only POST or you need to take both of them into account.